Makefile

```diff
@@ -7,14 +7,14 @@ install-cq:
 	chmod a+x cloudquery
 
 # start a timescale db running in a local container
-.PHONY: ts-start
-ts-start:
+.PHONY: start-ts
+start-ts:
 	docker run -p 5433:5432 -e POSTGRES_PASSWORD=pass -d timescale/timescaledb:latest-pg14
```

Comment:
So I actually found that `<action>-<db-flavor>` is harder to use than `<db-flavor>-<action>` because once you start working with a specific db-flavor you are going to want to continue to use it

Reply:
Good point! I'll update the PR to use the `<db-flavor>-<action>` format

Reply:
Done in https://github.com/cloudquery/cq-provider-aws/pull/763/commits/11daa6141b7b973937c461d9b7a2a4295695ece5